id/index.html

```diff
@@ -0,0 +1,371 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Accessibility Myths</title>
+    <link rel="stylesheet" href="./css/styles.css" />
+    <link rel="icon" type="image/x-icon" href="./favicon.ico" />
+
+    <meta name="description" content="A small project debunking common accessibility myths." />
+    <meta name="image" content="https://a11ymyths.com/" />
+
+    <meta property="og:title" content="Accessibility Myths" />
+    <meta property="og:description" content="A small project debunking common accessibility myths." />
+    <meta property="og:image" content="https://a11ymyths.com/images/og-image.png" />
+    <meta property="og:image:alt" content="Accessibility Myths Debunked" />
+    <meta property="og:url" content="https://a11ymyths.com/" />
+
+    <meta name="twitter:card" content="summary_large_image" />
+    <meta name="twitter:creator" content="@_sergeikriger" />
+    <meta name="twitter:title" content="Accessibility Myths" />
+    <meta name="twitter:description" content="A small project debunking common accessibility myths." />
+    <meta name="twitter:image" content="https://a11ymyths.com/images/og-image.png" />
+    <meta name="twitter:image:alt" content="Accessibility Myths Debunked" />
+  </head>
+  <body>
+    <main>
+      <div class="content">
+        <nav class="nav">
+          <ul class="nav__list">
+            <li class="nav__list-item">
+              <a class="nav__link" lang="ar" href="/ar/">العربية</a>
+            </li>
+            <li class="nav__list-item">
+              <a class="nav__link" lang="de" href="/de/">Deutsch</a>
+            </li>
+            <li class="nav__list-item">
+              <a class="nav__link" lang="en" href="/">English</a>
+            </li>
+            <li class="nav__list-item">
+              <a class="nav__link" lang="es" href="/es/">Español</a>
+            </li>
+            <li class="nav__list-item">
+              <a class="nav__link" lang="fr" href="/fr/">Français</a>
+            </li>
+            <li class="nav__list-item">
+              <a class="nav__link" lang="id" href="/id/">Bahasa Indonesia</a> 
+           </li>
```

Comment:
@franzaurus please add a link to Bahasa Indonesia to all languages (`fr/index.html`, `de/index.html`, etc.) 

<img width="963" alt="image" src="https://github.com/user-attachments/assets/b8cdadf0-23a0-448d-9225-aa2016109cb6">